src/journal/ObjectRecorder.cc

```diff
@@ -42,7 +42,13 @@ ObjectRecorder::ObjectRecorder(librados::IoCtx &ioctx, std::string_view oid,
   }
   m_compat_mode = require_osd_release < CEPH_RELEASE_OCTOPUS;
 
-  ldout(m_cct, 20) << dendl;
+  r = m_ioctx.stat(m_oid, &m_object_bytes, nullptr);
+  if (r < 0) {
+    ldout(m_cct, 5) << "failed to retrieve the size of object: "
```

Comment:
I'm wondering if this would spam the log due to `ENOENT` constantly being returned for new journal objects.

Reply:
Update to `r < 0 && r != -ENOENT`